examples/burial-form/ClaimantInformation.jsx

```diff
@@ -12,23 +12,23 @@ import { ExpandingGroupClass } from '../Constant';
 export default function ClaimantInformation(props) {
   const { values, setFieldValue } = useFormikContext();
   const getOptions = [
-    {label: "Spouse", value: "Spouse", key: 1},
-    {label: "Child", value: "Child", key: 2},
-    {label: "Parent", value: "Parent", key: 3},
-    {label: "Executor/Administrator of estate", value: "Executor/Administrator of estate", key: 4},
-    {label: "Other", value: "Other", key: 5},
+    {label: "Spouse", value: "spouse", key: 1},
+    {label: "Child", value: "child", key: 2},
+    {label: "Parent", value: "parent", key: 3},
+    {label: "Executor/Administrator of estate", value: "executorAdministratorEstate", key: 4},
+    {label: "Other", value: "other", key: 5},
```

Comment:
Is it required to submit these values as lowercase? Not sure if I understand the requirements in the original ticket @micahchiang

Reply:
@mlauren - It's sort of arbitrary to be honest. Values are decided by whichever app teams are implementing a thing and the business systems they are submitting to.